src/main/java/knightminer/inspirations/library/Util.java

```diff
@@ -80,10 +80,7 @@ public static ItemStack getStackFromState(IBlockState state) {
 
 		// first try getSilkTouchDrop, which just has to be protected
 		try {
-			Object stack = ReflectionHelper.findMethod(Block.class, "getSilkTouchDrop", "func_180643_i", IBlockState.class).invoke(block, state);
-			if(stack instanceof ItemStack) {
-				return (ItemStack) stack;
-			}
+			return ReflectionUtil.invokeGetSilkTouchDrop(block, state);
```

Comment:
Since this is abstracted out, it might be nice to not need the try/catch here, though I guess its a bit harder to tell "no result" from "failed to call method" then...